src/SignalR/clients/ts/signalr-protocol-msgpack/package.json

```diff
@@ -44,9 +45,7 @@
   ],
   "dependencies": {
     "@microsoft/signalr": "link:../signalr",
-    "msgpack5": "^4.5.0"
+    "@msgpack/msgpack": "^2.4.0"
```

Comment:
Just highlighting this to the SignalR team - not sure if you have any process to go through for the change of third-party dependency. @benaadams already updated the TPN file in this PR.

Reply:
We talked about this on the last DoI. We are planning to have a process to upgrade these dependencies to their latest versions on a regular basis, except for maybe the typescript compiler in the SignalR repo since we want to make sure it works with older version of the typescript compiler.

Reply:
> We talked about this on the last DoI

Well we didn't talk about changing the third-party library we use for some of the implementation.

Reply:
@BrennanConroy yeah, I was referring to updating the deps, changing the library you are right, that's up to you, but I think it's worth it.

Reply:
> Well we didn't talk about changing the third-party library we use for some of the implementation.

It is moving to the "official" msgpack lib under the msgpack organisation; rather than going the otherway, if that helps? https://github.com/msgpack/msgpack-javascript

Reply:
I think we are fine with this change, but I'll let @BrennanConroy confirm he's ok.